src/cryptoadvance/specter/templates/includes/tx-row.html

```diff
@@ -353,6 +353,7 @@
                         locked: this.tx.locked,
                     } });
                     this.dispatchEvent(event);
+                    e.stopPropagation(); // this prevent the onclick of outer divs to be triggered
```

Comment:
@relativisticelectron could you explain this? Which onclick listeneres picked that up? Where did that bubble to? I thought it would be contained within the web component.

Reply:
The entire row becomes clickable https://github.com/cryptoadvance/specter-desktop/blob/bb45ac6347b5fd96817fc8eb86d783f2437e1151/src/cryptoadvance/specter/templates/includes/tx-row.html#L206